src/Store/QuickmenuStoreConnector.re

```diff
@@ -160,8 +162,33 @@ let start = (themeInfo: ThemeInfo.t) => {
         Isolinear.Effect.none,
       )
 
-    | QuickmenuInputClicked(selection) => (
-        Option.map(state => Quickmenu.{...state, selection}, state),
+    | QuickmenuInputClicked((newSelection: Selection.t)) => (
+        Option.map(
+          (Quickmenu.{variant, selection, _} as state) => {
+            switch (variant) {
+            | Wildmenu(_) =>
+              let transition = selection.focus - newSelection.focus;
+
+              if (transition > 0) {
+                for (x in 0 to abs(transition)) {
+                  GlobalContext.current().dispatch(
+                    Actions.KeyboardInput("<LEFT>"),
+                  );
+                };
+              } else if (transition < 0) {
+                for (x in 0 to abs(transition)) {
```

Comment:
Don't need `abs` here either. You could either use `-transition`, or `downto` instead of `to`.